challenge.html

```diff
@@ -1,75 +1,88 @@
+        <section class= "steps">
+          <p class= "first_ind">For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+          <p class= "first_ind">Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+          <p class= "first_ind">Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+        </section>
         <img src="images/list-bg.png"
               alt="Recipe List" />
-        <link href='http://fonts.googleapis.com/css?family=Patrick+Hand' rel='stylesheet' type='text/css'>
+
         <ul class="ingredients">
-          <li>1 1/2 cups milk</li>
-          <li>1/2 cup mascarpone</li>
-          <li>1/2 tsp pink salt</li>
-          <li>1 lb Black Mission Figs</li>
-          <li>1/2 cup brown sugar</li>
-          <li>2-4 tbsp water</li>
-          <li>1 1/2 cups heavy cream</li>
-          <li>1/3 granulated sugar</li>
-          <li>2 egg yolks</li>
-          <li>1 lemon, juiced</li>
-          <li>2 tbsp butter</li>
-          <li>1 cup honey roasted pecans, roughly chopped</li>
+          <li><input type="checkbox" />1 1/2 cups milk</li>
+          <li><input type="checkbox" />1/2 cup mascarpone</li>
+          <li><input type="checkbox" />1/2 tsp pink salt</li>
+          <li><input type="checkbox" />1 lb Black Mission Figs</li>
+          <li><input type="checkbox" />1/2 cup brown sugar</li>
+          <li><input type="checkbox" />2-4 tbsp water</li>
+          <li><input type="checkbox" />1 1/2 cups heavy cream</li>
+          <li><input type="checkbox" />1/3 granulated sugar</li>
+          <li><input type="checkbox" />2 egg yolks</li>
+          <li><input type="checkbox" />1 lemon, juiced</li>
+          <li><input type="checkbox" />2 tbsp butter</li>
+          <li><input type="checkbox" />1 cup honey roasted pecans, roughly chopped</li>
         </ul>
     </article>
     <footer>
+      <img class= "greybar contain_16" src="images/grey_bar.png" />
       <img src="images/van-pic.png"
```

Comment:
I would consider the author info more as a part of the article than a part of the footer.